packages/components/src/internal/components/search/actions.ts

```diff
@@ -184,3 +189,69 @@ export function saveFinderGridView(schemaQuery: SchemaQuery, columns: any): Prom
         });
     });
 }
+
+export function saveFinderSearch(report: FinderReport, cardsJson: string, replace?: boolean): Promise<FinderReport> {
+    const reportConfig = {
+        name: report.reportName,
+        reportId: replace ? report.reportId : null,
+        config: cardsJson,
+        public: false,
+        replace,
```

Comment:
is replace saved with the report config, or is it enough to use it for the line 196 above?

Reply:
Replace is needed so we don't override another saved view with the same name when replace is false.